README.md

```diff
@@ -75,7 +75,7 @@ First you need to clone and build the cloud-hypervisor repo:
 $ pushd $CLOUDH
 $ git clone https://github.com/intel/cloud-hypervisor.git
 $ cd cloud-hypervisor
-$ cargo build --release
+$ ./install.sh
```

Comment:
I would be more in favor of leaving the `cargo build --release` as it is for now, and have the script being called from a `Pre-requisites` section. The script could be called `pre-reqs.sh` or `setup.sh` to ensure everything is present on the system.
But I would not mix the cargo build with this pre-req script since the pre-req step is needed only once, while the cargo build will be potentially called several times.